plugins/woocommerce/tests/e2e-pw/tests/merchant/create-shipping-zones.spec.js

```diff
@@ -90,6 +80,12 @@ test.describe( 'WooCommerce Shipping Settings - Add new shipping zone', () => {
 				'local_pickup'
 			);
 			await page.click( '#btn-ok' );
+			await page.waitForLoadState( 'networkidle' );
+			await expect(
+				page
+					.locator( '.wc-shipping-zone-method-title' )
+					.filter( { hasText: 'Local pickup' } )
+			).toBeVisible();
```

Comment:
There were cases where the shipping zone location erroneously was saved as "Everywhere" because the test navigates away from the page right after `await page.click( '#btn-ok' );` too soon. So I added a wait here, and an assertion, to make sure that the shipping zone location was saved as "Local pickup".

This isn't an issue much on `localhost` but our nightly smoke test site isn't fast enough to correctly save the shipping zone location if it immediately navigates away from the current page at the middle of saving it.

Reply:
I ran into that during the initial conversion, but could never quite nail down where "Everywhere" was coming from.  It seemed to resolve itself once we switched over to `wp-env` -- probably just because things were responsive enough!